apps.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+#IFS=':' read -r -a BASE_DIRS <<< "$XDG_DATA_DIRS"
```

Comment:
Mmm... does the fact that you needed `$HOME/.local/share` in `$XDG_DATA_DIR` have something to do with this script? (Not this line in particular, of course, I see that it's commented :-) )

Still... why? We have `$HOME` inside the container... if we know that there's stuff in `~/.local/share` that we need, we can just fetch it from there. What am I missing?

Reply:
This was a remnant of an original version of the script where it searched all XDG_DATA_DIRS for all icons and desktop files. It was a much slower process than the current implementation.